### PR TITLE
Try to fix imageio packaging for different python versions

### PR DESCRIPTION
**Pull request by mwperina**
Signed-off-by: Martin Perina <mperina@redhat.com>

### CODE REVIEW

**Comment by mwperina** [comment]
I tried to run COPR manual build and it seems to me that python requirements are correct for all packages now:

https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/build/5617540/

File: ovirt-imageio.spec.in [hunk 1]
```diff
@@ -224,6 +224,8 @@ Common resources used by oVirt imageio server and client
 %{python3_sitearch}/%{srcname}-*.egg-info
 %exclude %{python3_sitearch}/%{srcname}/client
 %exclude %{python3_sitearch}/%{srcname}/admin
+%define python3_pkgversion 3
+%define __python3 /usr/bin/python3
```
Comment (aesteve-rh) [inline ovirt-imageio.spec.in:228]: Aren't these overwriting the defines above in the same condition block?

**Review by mnecas — CHANGES_REQUESTED** [review]
-1 
looking at the daemon build on el8
it still requires the python3.9 
https://download.copr.fedorainfracloud.org/results/ovirt/ovirt-master-snapshot/centos-stream-8-x86_64/05617540-ovirt-imageio/ovirt-imageio-daemon-2.4.9-0.202303090719.git2f9c143.el8.x86_64.rpm

File: ovirt-imageio.spec.in (hunk 1 above)
Comment (mwperina) [reply ovirt-imageio.spec.in:228]: Those macros were needed only for `%{python3_sitearch}` usage above, so we change them back to the default value at the end. This should fix the issue with the order of package creation, which it not based on the position in specfile (see more details in OST chat).

**Pull request closed without merging**